File: TaskManagerSolution/src/main/java/com/taskmanager/model/Comment.java
package main.java.com.taskmanager.model;

import java.util.Date;
public class Comment {

    private String id;
    private String content;
    private String author;
    private Date createdAt;

    public Comment(String id,String content,String author,Date createdAt ) {
        this.id=id;
        this.content=content;
        this.author=author;
        this.createdAt=createdAt;
    }
}

File: TaskManager/TaskManager/Comment.cs
using System;

public class Comment
{
	public Comment()
	{
		public int ID;
		public string Content;
		public string Author;
		public dateTime CreatedAt;

        public Comment(int ID, string Content, string Author, dateTime CreatedAt)
        {
        this.ID = ID;
        this.Content = Content;
        this.Author = Author;
        this.CreatedAt = CreatedAt;
        }
}
